source/includes/_instruments.md

```diff
@@ -327,3 +327,165 @@ Http Status:
 | 400 Bad Request  | The request was malformed.  See response body    | None       | n/a              |
 | 401 Unauthorized | No auth provided, auth failed, or not authorized | None       | n/a              |
 | 404 Not Found    | No allocation with this UUID exists              | None       | n/a              |
+
+
+
+## Instrument Reporting
```

Comment:
My view would be to rename to 'Asset custody reconciliation'.

Reply:
And perhaps make it a section (e.g. `<em>Asset custody reconciliation</em>`)

Reply:
We could flesh this out more with instructions for the platform to submit on every working day before 9am for example (whilst still accepting on weekends / holidays).

Reply:
oki